### PR TITLE
Small kebab improvements

### PR DESCRIPTION
**Pull request by overlisted**
Added a simple bot example (+1 breaking change in mediasoup-client audio)

### CODE REVIEW

File: kebab/examples/bot/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          await wrapper.mutation.sendRoomChatMsg(stringToToken(Buffer.from(parameters, "base64").toString("utf-8")));
+
+          break;
+      }
+    });
+
+    console.info(`=> starting in room "${theRoom.name}" (${theRoom.numPeopleInside} people)`);
+    await wrapper.query.joinRoomAndGetInfo(theRoom.id);
+  } catch(e) {
+    if(e.code === 4001) console.error("invalid token!");
```
Comment (laforetcan) [inline kebab/examples/bot/src/index.ts]:
```suggestion
    if (e.code === 4001) console.error("invalid token!");
```

Comment (laforetcan) [reply kebab/examples/bot/src/index.ts]: Did you run prettier on the code?

**Review by tonoli — COMMENTED** [review]
Looks good to me except for prettier issues. Spaces after the if statements

File: kebab/examples/bot/src/index.ts (hunk 1 above)
Comment (overlisted) [reply kebab/examples/bot/src/index.ts:59]: No, I don't use prettier. I didn't even know that there has to be a space after `if`

**Pull request merged**